src/webui/www/public/scripts/dynamicTable.js

```diff
@@ -851,25 +851,33 @@ var TorrentsTable = new Class({
 
                 if (td.getChildren('div').length) {
                     var div = td.getChildren('div')[0];
-                    var newWidth = td.offsetWidth - 5;
-                    if (div.lastWidth !== newWidth) {
-                        div.setWidth(newWidth);
-                        div.lastWidth = newWidth;
+                    if (td.resized) {
+                        td.resized = false;
+                        div.setWidth(td._width - 5);
                     }
                     if (div.getValue() != progressFormated)
                         div.setValue(progressFormated);
                 }
-                else
+                else {
                     td.adopt(new ProgressBar(progressFormated.toFloat(), {
                         'width' : td.offsetWidth - 5
```

Comment:
300 lines of table need 10400ms
8500ms is needed for this line `'width' : td.offsetWidth - 5`
- 7800ms for rendering (calculate style) + 700ms for scripting

all other tasks need 1900ms
900ms layout
1000ms scripting